index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile Version Skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header-style">
+        <div class="logo">
+            
+           
+           <a href="#" class="logo">MY LOGO</a>
+            
+        </div>
+
+        <nav class="navbar">
+            <img src="Union.svg" alt="Navbar-Image" class="nav-image">
+        </nav>
+
+        
+            
+        
+    </header>
+
+    <section class="headling">
+        <img src="header-shapes-mobile.svg" alt="" class="header-shape-image">
+        <h1 class="intro">
+            I’m Mariane Glad to see you!
+        </h1>
+
+        <p class="professional-details">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+
+        <h4 class="social-media-intro">LET’S CONNECT</h4>
+
+        <section class="social-media-icon">
+            
+            <a class="list-style" href="#" > <img src="tweeter.svg"        alt="" ></a>
+            <a class="list-style" href="#" > <img src="Linkedin-icon.svg"  alt="" ></a>
+            <a class="list-style" href="#" > <img src="mediaThree.svg"     alt="" ></a>
+            <a class="list-style" href="#" > <img src="mediaFour.svg"      alt="" ></a>
+            <a class="list-style" href="#" > <img src="mediaFive.svg"      alt=""></a>   
+
```

Comment:
- [ ] Kindly implement the list of icons using a ul tag to group them and li per icon; each icon has a link (a tag) because those where the order of the items